src/foam/u2/tag/Input.js

```diff
@@ -137,8 +137,6 @@ foam.CLASS({
 
     function fromProperty(p) {
       this.SUPER(p);
-
-      if ( ! this.hasOwnProperty('onKey') && p.validateObj ) this.onKey = true;
```

Comment:
I don't think it is safe to remove this.

Reply:
We can't override the property onKey if we don't. The only thing that will work, is changing the call like for UnitValue in Elements.js

...
[ 'view', { class: 'foam.u2.view.CurrencyView' , onKey: false } ]